app/src/main/java/com/android/joinme/ui/overview/CreateSerieScreen.kt

```diff
@@ -0,0 +1,392 @@
+                          },
+                          colors =
+                              OutlinedTextFieldDefaults.colors(
+                                  disabledTextColor =
+                                      LocalContentColor.current.copy(
+                                          LocalContentColor.current.alpha),
+                                  disabledBorderColor = MaterialTheme.colorScheme.outline,
+                                  disabledLabelColor = MaterialTheme.colorScheme.onSurfaceVariant,
+                                  disabledPlaceholderColor =
+                                      MaterialTheme.colorScheme.onSurfaceVariant),
+                          singleLine = true)
+                    }
+
+                    val timePickerDialog = remember {
+                      android.app.TimePickerDialog(
+                          context,
+                          { _, selectedHour, selectedMinute ->
+                            val newTime = String.format("%02d:%02d", selectedHour, selectedMinute)
+                            createSerieViewModel.setTime(newTime)
+                          },
+                          hour,
+                          minute,
+                          true)
+                    }
+
+                    // Time field
+                    Box(modifier = Modifier.weight(1f).clickable { timePickerDialog.show() }) {
+                      OutlinedTextField(
+                          value = uiState.time,
+                          onValueChange = {},
+                          readOnly = true,
+                          enabled = false,
+                          label = { Text("Time") },
+                          placeholder = { Text("Select time") },
+                          modifier =
+                              Modifier.fillMaxWidth()
+                                  .testTag(CreateSerieScreenTestTags.INPUT_SERIE_TIME),
+                          isError = uiState.invalidTimeMsg != null,
+                          supportingText = {
+                            if (uiState.invalidTimeMsg != null) {
+                              Text(
+                                  text = uiState.invalidTimeMsg!!,
+                                  color = MaterialTheme.colorScheme.error)
+                            }
+                          },
+                          colors =
+                              OutlinedTextFieldDefaults.colors(
+                                  disabledTextColor =
+                                      LocalContentColor.current.copy(
+                                          LocalContentColor.current.alpha),
+                                  disabledBorderColor = MaterialTheme.colorScheme.outline,
+                                  disabledLabelColor = MaterialTheme.colorScheme.onSurfaceVariant,
+                                  disabledPlaceholderColor =
+                                      MaterialTheme.colorScheme.onSurfaceVariant),
```

Comment:
Same as lines 169 to 176 and lines 261 to 269

```suggestion
                          colors = readOnlyTextFieldColors(),
```